konrad/cloud.py

```diff
@@ -536,6 +546,89 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, coupling='pressure', **kwargs)
 
 
+class ConceptualCloud(DirectInputCloud):
+    def __init__(self, atmosphere, cloud_top, depth, cloud_fraction,
+                 water_path=100e-3, particle_size=100., phase='ice',
+                 coupling='pressure'):
+        """Initialize a grey cloud."""
```

Comment:
Could you please add the parameters to the docstring and say what units they should have (if applicable).

Reply:
Fair point! I forgot to add that after I finished prototyping

Reply:
Thanks! I find it weird that cloud depth is in pressure coordinates, which could definitely lead to some strange behaviour if the cloud moves too much. But if that's what you're using and it works for the cases you are investigating, it's fine!

Reply:
As far as I know, there is not obvious choice between fixed depth in height or pressure. I will just stick to the latter for simplicity. I guess it is one of the smaller problems when representing clouds in our 1D-RCE ;)
The only interesting simulation I would like to do at some point is fixing the cloud base temperature to turn-off that feedback.